example_impl/Dockerfile

```diff
@@ -12,4 +12,6 @@ COPY . .
 RUN go build -o kurtosis-go-example example_impl/main.go
 
 CMD set -euo pipefail && ./kurtosis-go-example \
-    --test-suite-container-id=${TEST_SUITE_CONTAINER_ID}
\ No newline at end of file
+    --test-names-filepath=${TEST_NAMES_FILEPATH} \
+    --test=${TEST} \
+    --kurtosis-api-ip=${KURTOSIS_API_IP} 2>&1 | tee ${LOG_FILEPATH}
```

Comment:
Can we add a newline here as well? I think there might be something off with your Goland settings, which is causing these to not be flagged during development